### PR TITLE
Churches

### PR DESCRIPTION
**Pull request by frett**
Add support for updating Churches

### CODE REVIEW

**Comment by frett** [comment]
@wrandall22 @mattfred 

To access update churches functionality you select the church on the map, then touch the info bubble that pops up.

Currently it only supports updating the contact name, email, and size of the church. But other fields can be added on pretty easily.

File: app/src/main/res/layout/fragment_edit_church.xml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        android:layout_gravity="center_horizontal"
+        android:layout_below="@+id/contactEmail"
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentStart="true"
+        tools:text="1234"/>
+
+    <Button
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Update"
```
Comment (mattfred) [inline app/src/main/res/layout/fragment_edit_church.xml]: Is this app going to be translated? Does this need to be in Strings?

Comment (frett) [inline app/src/main/res/layout/fragment_edit_church.xml]: yeah, I'll get that updated before this gets merged

**Comment by mattfred** [comment]
Are the edit text fields going to be given titles or descriptions?

**Comment by mattfred** [comment]
Looks good to me

**Comment by wrandall22** [comment]
Looks good to me

**Pull request merged**